content/docs/reference/tutorials/kubernetes/tutorial-migrate-nodegroups.md

```diff
@@ -0,0 +1,454 @@
+---
+title: "Clusters: EKS - Migrating Node Groups with Zero Downtime"
+menu:
+  reference:
+    parent: tutorials-kubernetes
+    weight: 3
+---
+
+In this tutorial we'll launch a new, managed Kubernetes cluster in Elastic
+Kubernetes Service (EKS) on AWS. Then, we'll walk through Day-2 operations for
+Kubernetes to help you administer and update the cluster, and your workloads running on it.
+
+We'll showcase how to:
+
+1. Create various worker node groups with different settings, such as instance types and AMIs.
+1. Deploy a workload to the set of node groups: the [NGINX Ingress Controller][ingress-nginx], and a
+simple [EchoServer][echoserver] app that echoes client request headers.
+1. Migrate NGINX from its original node group to a new, updated node
+group with zero downtime for both NGINX and the EchoServer during load testing.
+1. And lastly, once migration has completed, we'll drain, delete, and remove the
+original node group from Kubernetes and AWS.
+
+The code for this tutorial is available on [GitHub][example-gh].
+
+<p align="center"><img src="/images/docs/reference/kubernetes/eks-update-nodegroups.svg" width="650"></p>
+
+{{< aws-eks-prereqs >}}
+
+## Table of Contents
+
+- [Initialize the Pulumi Project](#initialize-the-pulumi-project)
+- [Create an EKS Cluster & Deploy Workloads](#create-an-eks-cluster-deploy-workloads)
+- [Access the Workloads](#access-the-workloads)
+- [...But First, Let's Talk About Resource Updates](#but-first-let-s-talk-about-resource-updates)
+  * [Pulumi's Approach: `create-before-delete`](#pulumi-s-approach-create-before-delete)
+  * [Kubernetes Workloads: High-Availability (HA) & Rolling Updates](#kubernetes-workloads-high-availability-ha-rolling-updates)
+- [The Great Migration](#the-great-migration)
+  * [Step 0: Launch Load Tests](#step-0-launch-load-tests)
+  * [Step 1: Creating the new `4xlarge` Node Group](#step-1-creating-the-new-4xlarge-node-group)
+  * [Step 2: Retargeting NGINX at the `4xlarge` Node Group](#step-2-retargeting-nginx-at-the-4xlarge-node-group)
+  * [Step 3: Decomissioning the `2xlarge` Node Group](#step-3-decomissioning-the-2xlarge-node-group)
+- [Clean Up](#clean-up)
+- [Summary](#summary)
+
+## Initialize the Pulumi Project
+
+1.  Start by cloning the [example][example-gh] to your local machine:
+
+    ```bash
+    git clone https://github.com/pulumi/pulumi-eks
+    cd nodejs/eks/examples/migrate-nodegroups
```

Comment:
Mind adding a `$` prefix in front of all these (applies throughout this doc)? We do that pretty consistently throughout the rest of the docs and there are even places in this doc that does it.

Also, the new "copy to clipboard" button will be smart about bash/shell lines prefixed with `$`... If these two lines were written as:

```bash
$ git clone https://github.com/pulumi/pulumi-eks
$ cd nodejs/eks/examples/migrate-nodegroups
```

And you click the copy button, the following will be written to the clipboard:

```bash
git clone https://github.com/pulumi/pulumi-eks && cd nodejs/eks/examples/migrate-nodegroups
```

If you had comments (starting with `#`) or lines that came after lines that are prefixed with `$ ` (e.g. command output), those won't be written to the clipboard.

Reply:
Done! PS This smart copy feature + the copy feature itself is a great add! 🎉 Thanks for the tip